terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.test.ts

```diff
@@ -669,7 +669,20 @@ runner_types:
       os: linux
       max_available: 1
       disk_size: 150
-      is_ephemeral: false`;
+      is_ephemeral: false
+    linux.4xlarge:
+      instance_type: c5.2xlarge
+      os: linux
+      max_available: 1
+      disk_size: 150
+      is_ephemeral: false
+      variants:
```

Comment:
as a naming nit: thoughts on calling these "experiments" to emphasize that they're meant to be used for temporary variations instead of permanent runner-types?

Reply:
Not sure, bc we might want to eventually have variants that are fixed, I don't want to tie this to a specific and unique UC